I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 123, in script
    play sounds sfx_qanda fadein 0.1
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
Exception: Audio channel 'sounds' is unknown.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 123, in script
    play sounds sfx_qanda fadein 0.1
  File "C:\Users\vanes\Downloads\Setups\renpy-8.0.3-sdk\renpy\ast.py", line 2232, in execute
    self.call("execute")
  File "C:\Users\vanes\Downloads\Setups\renpy-8.0.3-sdk\renpy\ast.py", line 2220, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\vanes\Downloads\Setups\renpy-8.0.3-sdk\renpy\statements.py", line 281, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "C:\Users\vanes\Downloads\Setups\renpy-8.0.3-sdk\renpy\audio\music.py", line 103, in play
    c = get_channel(channel)
  File "C:\Users\vanes\Downloads\Setups\renpy-8.0.3-sdk\renpy\audio\audio.py", line 915, in get_channel
    raise Exception("Audio channel %r is unknown." % name)
Exception: Audio channel 'sounds' is unknown.

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090809
Root of Evil 1.0
Sat Feb  4 15:00:27 2023
